motr/client.h

```diff
@@ -917,8 +919,8 @@ enum m0_pi_calc_flag {
 
 	/* PI calculation for data unit 0 */
 	M0_PI_CALC_UNIT_ZERO = 1 << 0,
-	/* PI final value to be calculated */
-	M0_PI_CALC_FINAL = 1 << 1
+	/* Skip PI final value calculation */
+	M0_PI_SKIP_CALC_FINAL = 1 << 1
```

Comment:
@gshipra @nileshgovande  What should i pass if i dont want either M0_PI_CALC_UNIT_ZERO or M0_PI_SKIP_CALC_FINAL. 
I was going to pass 0, but the param is an enum not an integer. If i cast 0 to the enum it can produce undefined behavior as per C++11 and C++14 Standards 

> A value of integral or enumeration type can be explicitly converted to an enumeration type. The value is unchanged if the original value is within the range of the enumeration values (7.2). Otherwise, the resulting value is unspecified (and might not be in that range).

Reply:
You can pass 0. I have passed 0 in my UT code and it did not give any warning or unwanted result.